src/scripts/main.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
 function waitFor(element, eventName) {
-// write your code here
+  return new Promise((resolve) => {
+    element.addEventListener(eventName, () => {
+      resolve(`
+            It was ${eventName} on the element:
```

Comment:
Consider removing the whitespace at the beginning of the string for better readability.